pango/sys/src/lib.rs

```diff
@@ -924,9 +952,10 @@ impl ::std::fmt::Debug for PangoGlyphString {
 }
 
 #[repr(C)]
-#[derive(Copy, Clone)]
 pub struct PangoGlyphVisAttr {
     pub is_cluster_start: c_uint,
+    _truncated_record_marker: c_void,
+    // field is_color has incomplete type
```

Comment:
I'm not sure what it means to have an incomplete type, it seems to be correct here
```xml
    <record name="GlyphVisAttr" c:type="PangoGlyphVisAttr">
      <doc xml:space="preserve">A `PangoGlyphVisAttr` structure communicates information between
the shaping and rendering phases.

Currently, it contains cluster start and color information.
More attributes may be added in the future.</doc>
      <field name="is_cluster_start" writable="1" bits="1">
        <doc xml:space="preserve">set for the first logical glyph in each cluster.
  (Clusters are stored in visual order, within the cluster, glyphs
  are always ordered in logical order, since visual order is meaningless;
  that is, in Arabic text, accent glyphs follow the glyphs for the
  base character.)</doc>
        <type name="guint" c:type="guint"/>
      </field>
      <field name="is_color" writable="1" bits="1">
        <doc xml:space="preserve">set if the the font will render this glyph with color. Since 1.50</doc>
        <type name="guint" c:type="guint"/>
      </field>
    </record>
```

Reply:
Because it's a bit field and bit fields are difficult (and behave different between compilers, etc). The support for them in our code generator is very minimal.